src/pseudopeople/dataset.py

```diff
@@ -47,7 +47,7 @@ def __init__(
     def __bool__(self):
         return not self.data.empty
 
-    def is_empty(self, column_name: str) -> bool:
+    def is_empty(self, column_name: str) -> pd.BooleanDtype:
```

Comment:
What is `pd.BooleanDtype`? Are we sure we don't want to convert this to an actual `bool` here?

Reply:
That got the original error to work but I have reverted it to declare it as a bool like we do with other types elsewhere to stay consistent